encoding/data.go

```diff
@@ -41,8 +41,9 @@ func (f *Frame) Length() int {
 }
 
 // Returns the size of chunk in bytes.
-func (f *Frame) Size() int {
-	return f.Length() * BYTES_PER_COEFFICIENT
+func (f *Frame) Size() uint64 {
+	proofSize := uint64(2 * 32)
```

Comment:
This may make the blobs stored and removed not the same at Node.
When storing blobs, it's in this entry: https://github.com/Layr-Labs/eigenda/blob/4db2480f472fc6958786ffc95eea96b39fee2d20/node/store.go#L246
When removing blobs, it's deducting this number: https://github.com/Layr-Labs/eigenda/blob/4db2480f472fc6958786ffc95eea96b39fee2d20/node/store.go#L142

Reply:
ah ok i'll exclude the proof size and comment that this method only returns the size of the chunks